.github/workflows/docker.yml

```diff
@@ -10,13 +10,18 @@ jobs:
     env:
       PUPPERWARE_ANALYTICS_STREAM: production
       IS_LATEST: true
+      # Still need these env vars for README publishing
       DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
       DOCKERHUB_PASSWORD: ${{ secrets.DOCKERHUB_PASSWORD }}
 
     runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@master
+      - uses: azure/docker-login@v1
+        with: # This doesn't seem to work unless we point directly to the secrets
```

Comment:
Yeah, I don't see anything special in the code - https://github.com/Azure/docker-login/blob/master/src/login.ts